app/services/flood_risk_engine/find_address_by_postcode.rb

```diff
@@ -0,0 +1,29 @@
+module FloodRiskEngine
+  class FindAddressByPostcode
+
+    attr_reader :address_result_set
+
+    def initialize(post_code)
+      Rails.logger.debug("FindAddressByPostcode supplied with params [#{post_code.inspect}]")
+      @post_code = post_code
+      @address_result_set = []
+    end
+
+    def search
+      @address_result_set = begin
+        EA::AddressLookup.find_by_postcode(post_code)
+      rescue => ex
+        Airbrake.notify(ex) if defined? Airbrake
+        Rails.logger.error "Address lookup service failed: #{ex}"
+        []
+      end
+
```

Comment:
ok I'll look into that. I think originally the rescue had to be there to report properly to Airbrake